packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -323,6 +350,22 @@ const DataSeriesFormItemModal = ({
                 </span>
               </div>
             )}
+            {editDataItem?.hasStreamingMetricEnabled && onAddAggregations && (
+              <div className="addAggregation">
```

Comment:
let's prefix this and use BEM style for classnames please
```suggestion
              <div className={`${iotPrefix}--add-aggregation`}>
```

Reply:
done